.vscode/settings.json

```diff
@@ -70,20 +70,9 @@
     "editor.formatOnSave": false
   },
 
-  "python.linting.pylintEnabled": false,
-  "python.linting.flake8Enabled": true,
-  // test discovery is sluggish and the UI around running
-  // tests is often in your way and misclicked
+  "editor.tabSize": 4,
   "python.testing.pytestEnabled": true,
   "python.testing.unittestEnabled": false,
-  "python.testing.nosetestsEnabled": false,
-  "editor.tabSize": 4,
```

Comment:
I don't believe this setting is deprecated, at least when I look at the `settings.json` in my editor.

Reply:
OH ITS MOVED oops lol

Reply:
yeah just moving it out of the middle